libraries/botbuilder-dialogs/tests/attachmentPrompt.test.js

```diff
@@ -57,10 +57,9 @@ describe('AttachmentPrompt', function() {
 
         const dialogState = convoState.createProperty('dialogState');
         const dialogs = new DialogSet(dialogState);
-        dialogs.add(new AttachmentPrompt('prompt', (context, prompt) => {
-            assert(context);
+        dialogs.add(new AttachmentPrompt('prompt', async (prompt) => {
             assert(prompt);
-            prompt.end(prompt.recognized.value);
+            return prompt.recognized.succeeded;
```

Comment:
@v-ducvo, @kaiqb, here's a delta in how to use validators that needs to be documented.